src/Command/RenderBlockCommand.php

```diff
@@ -78,13 +125,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $output->writeln("\n<info>Response Output</info>");
 
         // fake request
-        $request = new Request();
-        $this->getContainer()->enterScope('request');
-        $this->getContainer()->set('request', $request, 'request');
-
-        $output->writeln($this->getContainer()->get('sonata.block.renderer')->render($context));
-
-        $this->getContainer()->leaveScope('request');
```

Comment:
`enterScope`/`leaveScope` functions do not exist anymore, I don't know how to reproduce this behaviour and I'm not sure what the goal was.

Reply:
I have no idea ^^

Reply:
This is for when request was an object on the container, We should ensure no one tries to access request directly from the container and use requestStack instead.

Reply:
[Scopes are deprecated since Symfony 2.8](https://symfony.com/blog/new-in-symfony-2-8-deprecating-scopes-and-introducing-shared-services), is there anything to add here to keep the same behaviour as before?

Reply:
I would say not